sound/soc/sof/pcm.c

```diff
@@ -281,7 +281,6 @@ static int sof_pcm_trigger(struct snd_soc_component *component,
 	struct snd_sof_dev *sdev = snd_soc_component_get_drvdata(component);
 	const struct sof_ipc_pcm_ops *pcm_ops = sof_ipc_get_ops(sdev, pcm);
 	struct snd_sof_pcm *spcm;
-	bool reset_hw_params = false;
```

Comment:
The flag was added for a reason, at least the commit message was very specific on what this is suppose to handle:
04c8027764bc ("ASoC: SOF: reset DMA state in prepare")

It might lost it's purpose and relevance over time, but aren't we going to break something with this?

@kv2019i ?

Reply:
@ujfalusi allot of has changed in terms of how we free the widgets with all.thenwork around ipc4. And if you think about it, platform_hw_params gets the stream tag and if with start/stop tests,.there's no setting up widgets again, we need preserve the original tag.